index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="styles.css" />
+    <title>Teacher's P.E.T.</title>
+  </head>
+  <body>
+    <!-- div for user menu -->
+    <div id="containerUsers">
+      <h2>Teachers P.E.T.</h2>
+      <div id="users">
+        <nav id="navWrap" class="">
+          <ul>
+            <li a href="#" class="" id="menuLabel">User</li>
+            <ul id="submenu">
+              <li class="userBtn" id="teacher">
+                <a href="#">Teacher</a></li>
+              <li class="userBtn" id="student">
+                <a href="#">Student</a></li>
+              <li class="userBtn" id ="parent">
+                <a href="#">Parent</a></li>
+            </ul>
+          </ul>
+        </nav>
+      </div>
+    </div>
+    <!-- div for the teacher content -->
+    <div id="teacherCont" class="hidden">
+      <div id=""
```

Comment:
This is div is invalid. It is missing the closing `>` and it closing tag pair `</div>`.